### ISSUE TITLE
Crash upon trying to create a new folder

### ISSUE THREAD

**Issue by bbaserdem** [SamSchott/maestral-dropbox#31]
I'm on Linux, and have the following error on the stable release (0.2.3) when i try to set the dropbox folder using gui.

```

Gkr-Message: 17:08:09.928: secret service operation failed: The name org.freedesktop.secrets was not provided by any .service files
 > Loading access token...
 > MaestralClient is ready.
Up to date
Maestral index could not be found. Rebuild if necessary.
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/maestral/gui/first_sync_dialog.py", line 200, in on_dropbox_path
    self.populate_folders_list()
  File "/usr/lib/python3.7/site-packages/maestral/gui/first_sync_dialog.py", line 241, in populate_folders_list
    root_folders = self.mdbx.client.list_folder("", recursive=False)
  File "/usr/lib/python3.7/site-packages/maestral/client.py", line 476, in list_folder
    **kwargs
TypeError: files_list_folder() got an unexpected keyword argument 'include_non_downloadable_files'
/usr/bin/maestral-gui: line 6: 16289 Aborted
```

I assume has to do with python-dropbox, which is 9.2.0

**Comment by bbaserdem** [comment]
I was able to fix this by using python 9.4.0 through pip.

Since this is an issue with the packager, i will close this as solved; as official installation instructions don't lead to this error.

**Comment by SamSchott** [comment]
Apologies, I forget to require the newest dropbox SDK package in setup.py. This is fixed in 913107466e4113e9f7f4d54a535098540b2aedbb and will be rolled out in v0.2.4 of Maestral.
